aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -233,7 +233,12 @@ void gemm<at::Half>(CUDABLAS_GEMM_ARGTYPES(at::Half)) {
 #else
   cudaDeviceProp* prop = at::cuda::getCurrentDeviceProperties();
   if (prop->major >= 5) {
+#if defined(CUDA_VERSION) && CUDA_VERSION < 11000
+    // On Cuda 11, tensor cores are allowed by default for FP16 inputs if the math mode is
```

Comment:
This check and the comment are mismatched. The check is what do when using a version of CUDA before 11, but the comment describes what should be done (or not done) on CUDA 11 (and, on future CUDAs?). 

I would have expected this comment, instead, to start with something like: "On CUDA versions prior to 11, ..."

Reply:
fixed